packages/commands/src/limeobject/create-dialog.command.ts

```diff
@@ -33,4 +33,9 @@ export class CreateLimeobjectDialogCommand {
      * Title of the dialog
      */
     public title?: string;
+
+    /*
+     * Current context
+     */
+    public context?: string;
```

Comment:
This should not be a string, right? 
```suggestion
    public context?: LimeWebComponentContext;
```

Reply:
😮  Great catch!